proto/cosmos/staking/query.proto

```diff
@@ -7,222 +7,261 @@ import "cosmos/staking/staking.proto";
+  // validator_addr defines the validator address to query for.
   bytes validator_addr = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 }
 
 // QueryValidatorResponse is response type for the Query/Validator RPC method
 message QueryValidatorResponse {
+  // validator defines the the validator info.
   Validator validator = 1 [(gogoproto.nullable) = false];
 }
 
 // QueryValidatorDelegationsRequest is request type for the Query/ValidatorDelegations RPC method
 message QueryValidatorDelegationsRequest {
+  // validator_addr defines the validator address to query for.
   bytes validator_addr = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 
+  // pagination defines an optional pagination for the request.
   cosmos.query.PageRequest pagination = 2;
 }
 
 // QueryValidatorDelegationsRequest is response type for the Query/ValidatorDelegations RPC method
 message QueryValidatorDelegationsResponse {
   repeated DelegationResponse delegation_responses = 1 [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "DelegationResponses"];
 
+  // pagination defines the pagination in the response.
   cosmos.query.PageResponse pagination = 2;
 }
 
 // QueryValidatorUnbondingDelegationsRequest is required type for the Query/ValidatorUnbondingDelegations RPC method
 message QueryValidatorUnbondingDelegationsRequest {
+  // validator_addr defines the validator address to query for.
   bytes validator_addr = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 
+  // pagination defines an optional pagination for the request.
   cosmos.query.PageRequest pagination = 2;
 }
 
-// QueryValidatorUnbondingDelegationsResponse is response type for the Query/ValidatorUnbondingDelegations RPC method
+// QueryValidatorUnbondingDelegationsResponse is response type for the Query/ValidatorUnbondingDelegations RPC method.
 message QueryValidatorUnbondingDelegationsResponse {
   repeated UnbondingDelegation unbonding_responses = 1 [(gogoproto.nullable) = false];
 
+  // pagination defines the pagination in the response.
   cosmos.query.PageResponse pagination = 2;
 }
 
-// QueryDelegationRequest is request type for the Query/Delegation RPC method
+// QueryDelegationRequest is request type for the Query/Delegation RPC method.
 message QueryDelegationRequest {
+  // delegator_addr defines the delegator address to query for.
   bytes delegator_addr = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
 
+  // validator_addr defines the validator address to query for.
   bytes validator_addr = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 }
 
-// QueryDelegationResponse is response type for the Query/Delegation RPC method
+// QueryDelegationResponse is response type for the Query/Delegation RPC method.
 message QueryDelegationResponse {
-  DelegationResponse delegation_response = 1 [(gogoproto.casttype) = "DelegationResponse"];
```

Comment:
Removed useless casttype here. Saw a couple of other useless casttype in other files (genesis), will take care of that in future PRs.